subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -56,21 +51,19 @@ static void comp_data_status(struct bt_mesh_model *model,
 	       ctx->net_idx, ctx->app_idx, ctx->addr, buf->len,
 	       bt_hex(buf->data, buf->len));
 
-	if (!cli_response_check(OP_DEV_COMP_DATA_STATUS, ctx->addr)) {
-		BT_WARN("Unexpected Composition Data Status");
+	if (!bt_mesh_msg_ack_ctx_match(&cli->ack_ctx, OP_DEV_COMP_DATA_STATUS, ctx->addr,
+				       (void **) &param)) {
```

Comment:
Is the explicit typecast really needed? Do you get some warning without it? Anyway, if it can't be avoided the coding style is to not have a space before the variable name.

Reply:
Yeah, I get warning here. Will remove the space.